src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const buttons = document.querySelectorAll('.button');
+const table = document.querySelector('tbody');
+const container = document.querySelector('.container');
+let rowsAmnt = table.rows.length;
+let colAmnt = table.rows[0].cells.length;
+
+container.addEventListener('click', e => {
+  const target = e.target;
+
+  if (target.classList.contains('append-row')) {
+    const row = table.rows[0].cloneNode(true);
+
+    table.append(row);
+    rowsAmnt++;
+  };
+
+  if (target.classList.contains('remove-row')) {
+    table.rows[rowsAmnt - 1].remove();
+    rowsAmnt--;
+  };
+
+  if (target.classList.contains('append-column')) {
+    for (const el of table.rows) {
+      el.append(document.createElement('td'));
+    };
+    colAmnt++;
+  };
+
+  if (target.classList.contains('remove-column')) {
+    for (const el of table.rows) {
+      el.cells[colAmnt - 1].remove();
+    };
+    colAmnt--;
+  };
```

Comment:
This code is similar. You can use the auxiliary function. Think about how you can combine them and pass parameters to the function